astropy/wcs/wcsapi/tests/test_fitswcs.py

```diff
@@ -587,31 +587,31 @@ def test_time_1d_values_time(header_time_1d):
     assert_time_at(header_time_1d, 1, 2450003, 0.1 + 7 / 3600 / 24, 'tai', 'mjd')
 
 
-@pytest.mark.remote_data
-@pytest.mark.parametrize('scale', ('tai', 'tcb', 'tcg', 'tdb', 'tt', 'ut1', 'utc'))
-def test_time_1d_roundtrip(header_time_1d, scale):
-
-    # Check that coordinates round-trip
-
-    pixel_in = np.arange(3, 10)
-
-    header_time_1d['CTYPE1'] = scale.upper()
-
-    with warnings.catch_warnings():
-        warnings.simplefilter('ignore', FITSFixedWarning)
-        wcs = WCS(header_time_1d)
-
-    # Simple test
-    time = wcs.pixel_to_world(pixel_in)
-    pixel_out = wcs.world_to_pixel(time)
-    assert_allclose(pixel_in, pixel_out)
-
-    # Test with an intermediate change to a different scale/format
-    time = wcs.pixel_to_world(pixel_in).tdb
-    time.format = 'isot'
-    pixel_out = wcs.world_to_pixel(time)
-    assert_allclose(pixel_in, pixel_out)
-
+# @pytest.mark.remote_data
+# @pytest.mark.parametrize('scale', ('tai', 'tcb', 'tcg', 'tdb', 'tt', 'ut1', 'utc'))
+# def test_time_1d_roundtrip(header_time_1d, scale):
```

Comment:
Why is this test commented?

Reply:
Oops, my mistake. Thanks for catching this.